tdrs-frontend/.eslintrc.json

```diff
@@ -24,7 +24,6 @@
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx", ".ts", ".tsx"] }],
     "import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
     "react/jsx-props-no-spreading": ["off"],
-    "react/button-has-type": ["off"],
```

Comment:
This was needed when I was experimenting with `react-select` but is no longer needed.